docs/reference/alpha-features.md

```diff
@@ -23,25 +27,23 @@ Characteristics of alpha features and components include:
 - (SaaS) No availability targets.
 - Released outside the standard [release policy](/reference/release-policy.md).
 
-:::note
-
-While there is no maintenance service, customers can still provide feedback through designated support channels, depending on their SLAs. These channels include filing issues in the respective [GitHub repositories](https://github.com/camunda) and by submitting questions and suggestions by [contacting us](/contact).
+To learn more about using alpha features, see [enabling alpha features](/components/console/manage-organization/enable-alpha-features.md).
 
+:::note
+Although there is no maintenance service, customers can still provide feedback through designated support channels, depending on their SLAs. These channels include filing issues in the respective [GitHub repositories](https://github.com/camunda) and by submitting questions and suggestions by [contacting us](/contact).
 :::
 
-Visit our documentation on [enabling alpha features](/components/console/manage-organization/enable-alpha-features.md) to learn more.
-
 ### General availability (GA)
 
 Once features and components are released and considered stable, they become generally available.
 
-Characteristics include:
+Stable features and components are:
 
 - Allowed for production use.
 - Fully documented and supported.
 
-:::note
-
-There are also alpha releases with **limited availability**, such as features that are only available to enterprise customers.
+A release or component is considered stable if it has passed all stages of verification and tests and can be released to production.
```

Comment:
Why do we have this "note" - what value does it add? I would remove it

Reply:
@MaxTru I guess this is to cover us for any customers who could be annoyed if they do not get access to an alpha release? I'd be inclined to leave it as there must have been a reason for this initially (perhaps prompted by a customer/legal), unless you strongly want to remove it?